sysutils/calcurse/Portfile

```diff
@@ -1,7 +1,7 @@
 PortSystem 1.0
```

Comment:
@contextnerror, can you add a modeline?  The first line of any Portfile should ideally be the modeline (ref: https://guide.macports.org/#development.creating-portfile )

```suggestion
# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
PortSystem 1.0
```